src/sinks/clickhouse/config.rs

```diff
@@ -95,3 +124,30 @@ impl SinkConfig for ClickhouseConfig {
         &self.acknowledgements
     }
 }
+
+async fn healthcheck(client: HttpClient, endpoint: Uri, auth: Option<Auth>) -> crate::Result<()> {
+    // TODO: check if table exists?
```

Comment:
Probably out of scope, but consider addressing this TODO now - or if you're going to tackle the template-able table next maybe do it then given it'd be impacted by the change.

Reply:
Once we add template-able tables, this TODO is no longer possible because the table will be dynamic. I'll remove it then.